File: docs/source/_src_snippets/simulation/propagation_setup/acceleration_models/spherical_harmonic_gravity.py

maximum_degree = 12
maximum_order = 12

accelerations_on_vehicle = dict(
            Earth = [ propagation_setup.Acceleration.spherical_harmonic_gravity(maximum_degree, maximum_order) ] 
        )


accelerations = {"Apollo": accelerations_on_vehicle}

acceleration_models = propagation_setup.create_acceleration_models_dict(
            bodies, accelerations, bodies_to_propagate, central_bodies)

File: docs/source/_src_snippets/simulation/propagation_setup/acceleration_models/spherical_harmonic_gravity.cpp
SelectedAccelerationMap accelerationSettings;

int maximumDegree = 12;
int maximumOrder = 12;

accelerationSettings[ "Apollo" ][ "Earth" ].push_back( 
	std::make_shared< SphericalHarmonicAccelerationSettings >( maximumDegree, maximumOrder ) );